rules-tests/Php71/Rector/FuncCall/RemoveExtraParametersRector/Fixture/skip_phpstan_phar.php.inc

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Rector\Tests\Php71\Rector\FuncCall\RemoveExtraParametersRector\Fixture;
```

Comment:
You can create different directory for this fixture ,eg: `FixtureAutoImport`, then next:

1. add the config for this, eg: `config/configured_rule_auto_import.php`
2. add a test class, eg: `AutoImportTest.php` that refer to new `FixtureAutoImport` directory and `config/configured_rule_auto_import.php`

You can refer the structure like this: 

```
├── RemoveExtraParametersRectorTest.php
├── AutoImportTest.php
├── Fixture
├── FixtureAutoImport
│   └── skip_phpstan_phar.php.inc
├── Source
└── config
    ├── configured_rule_auto_import.php
    ├── configured_rule.php

```

Reply:
The test for `phpstan.phar` is exclusive to the `RemoveExtraParametersRector`. So that makes me think this test is in the right place?

The fix in `ReflectionResolver.php` for `ShortenedObjectType` is more generic. As I said, I don't know how to test that one as I don't know when a `ShortenedObjectType` occurs. I can split it into 2 separate PRs if you want?

Reply:
Better to have failing test case first commited to see the error, and then commit the fix so we can see the different, separate PR for smaller PR will be ok